### ISSUE TITLE
Cannot enable RTL-SDR 

### ISSUE THREAD

**Issue by ngmacha** [jopohl/urh#224]
(This is copied from #220 for follow up)

Initial Issue: Somehow the Devices has only RTL-TCP enabled, and all other devices options are greyed out. I enabled it uder ~/.config/urh.ini, but starting reset everything back to disabled. I'm sure that my RTL-SDR is correctly installed, as gqrx is running fine. But do I need gnuradion to be able to use it under URH, as I was unable to install the full gnuradio yet?

@jopohl : For devices you should ensure that you have the required libs installed on your system. What does locate rtlsdr.so say? You should not enable devices manually in ini. Native backend should work for you if you installed rtlsdr.so, you can use the rebuild button in device options for this.

**Comment by ngmacha** [comment]
I used `sudo find / -name *rtlsdr.so` as my system doesn't have the `locate` command. It found the `librtlsdr.so` under `rtl-sdr/build/src/ folder`, as I've built the RTL-SDR from source. 

I've copied two files (was not sure if both are needed?) `librtlsdr.so` and `lirtlsdr.so.0` to `/usr/lib/rtl-sdr`, entered that path into the Device tab and clicked Rebuild and the RTL-SDR now can be enabled.

I did the same for HackRF (the files `libhackrf.so` and `libhackrf.so.0` were under `hackrf/host/build/libhackrf/src`) 

The issue remains now, that every time I start URH, I have to go to Options->Device and re-enable it again, as the setting is not retained.
